docker-compose.yml

```diff
@@ -40,5 +41,6 @@ services:
       - AB2D_DB_USER=ab2d
       - AB2D_DB_PASSWORD=ab2d
       - AB2D_EFS_MOUNT=/tmp/ab2d_efs_mount
+      - SPRING_PROFILE=prod
```

Comment:
It is a bit odd to default the value of spring profile to `PROD`. 
I wonder if we should default it to some other environment? like `dev` or `qa` or something which is _not prod_ ?

Reply:
Hmm I'm not sure if we have those environments up at the moment, currently there's still not any true prod environment so I think this can be revisited once some more decisions come. In the future we might have a docker-compose-qa.yml or some other way of changing environments.